sunpy/timeseries/timeseriesbase.py

```diff
@@ -68,25 +68,21 @@ class GenericTimeSeries:
     """
     A generic time series object.
 
+    Parameters
+    ----------
+    data : `~pandas.DataFrame`
+        A pandas DataFrame representing one or more fields as a function
+        of time.
+    meta : `~sunpy.timeseries.metadata.TimeSeriesMetaData`, optional
+        The metadata giving details about the time series data/instrument.
+
```

Comment:
units should be added in both parameters and attributes. Units are a dictionary of column name : unit .

Reply:
This is the final issue before merging?